Beginner_Projects/Currency-Converter-with toggletheme/currency_converter.py

```
@@ -0,0 +1,173 @@
+import tkinter as tk
+import requests # type: ignore
+from tkinter import messagebox
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+API_KEY =os.getenv("API_KEY")
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)